CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -65,42 +65,44 @@ class HomeViewController: UIViewController {
         view.addSubview(tabCollectionView)
 
         setUpConstraints()
+        setUserAndTabPage()
     }
 
-    private func getUserMatching() {
+    private func setUserAndTabPage() {
+        getUserThen { user in
+            self.user = user
+
+            if let pictureURL = URL(string: user.profilePictureURL) {
+                self.profileButton.kf.setImage(with: pictureURL, for: .normal)
+            }
+
+            let firstActiveMatch = user.matches.filter({ $0.status != "inactive" }).first
```

Comment:
Is the backend giving inactive matches too? :o or multiple matches?

Reply:
I was using the user response endpoint to get all the info at once for the user and matches, and there is no endpoint that gets just one match including a user atm